db/schema.rb

```diff
@@ -46,10 +46,11 @@
     t.string "action_link_url"
     t.string "action_link_content"
     t.string "action_link_summary"
+    t.string "sub_heading"
```

Comment:
Looks like you've got this from your new migration. You'll have to roll that back first (or do db:reset on the main branch if you're not precious about the data)